### ISSUE TITLE
[With log] Cannot build from dock/lauchpad/survey stake

### ISSUE THREAD

**Issue by chengxinlun** [taniwha/Extraplanetary-Launchpads#131]
I first encountered this problem when working on a Mum base with KPBS, and I have tested stock workshop+dock/launchpad and it also failed. Basically, when I clicked the finalize build button nothing happens.

I have checked the log and seems to locate the problem. Clicking the finalize button throws ```MissingMethodException: Method not found: 'Vessel.GetHeightFromTerrain'.``` The ship I am going to build, ```KIT Mk2```, makes completely out of stock parts.

```
[LOG 15:51:57.154] KIT Mk2 loaded!
[LOG 15:51:57.159] []: Ready to Launch - waiting to start physics...
[LOG 15:51:57.172] [IFS] - InsterstellarFuelSwitch SetupTankInPart adding resources: LiquidFuel;Oxidizer;
[LOG 15:51:57.175] [KSPI] - Creating Resource Manager for Vessel  (Megajoules)
[LOG 15:51:57.176] -INFO- Tac.LifeSupportController[FF876F52][9889.20]: Vessel Destroyed [00000000-0000-0000-0000-000000000000] 
[LOG 15:51:57.176] [PR] Deleting  as reference.
[LOG 15:51:57.176] [PR] Generating data for 
[WRN 15:51:57.183] [KSPI] - detecting supplyable part SP-L 1x6 Photovoltaic Panels is being destroyed
[WRN 15:51:57.183] [KSPI] - detecting supplyable part SP-L 1x6 Photovoltaic Panels is being destroyed
[WRN 15:51:57.183] [KSPI] - detecting supplyable part SP-L 1x6 Photovoltaic Panels is being destroyed
[WRN 15:51:57.183] [KSPI] - detecting supplyable part SP-L 1x6 Photovoltaic Panels is being destroyed
[LOG 15:51:57.199] 10/14/2017 3:51:57 PM,KerbalAlarmClock,Triggering Alarm - [EL] build: "KIT Mk2"
[LOG 15:51:57.199] 10/14/2017 3:51:57 PM,KerbalAlarmClock,[EL] build: "KIT Mk2"-Halt Warp
[LOG 15:51:57.201] 10/14/2017 3:51:57 PM,KerbalAlarmClock,Actioning Alarm
[LOG 15:51:57.201] 10/14/2017 3:51:57 PM,KerbalAlarmClock,KillWarp-Yes-False-False
[LOG 15:52:01.290] Packing Untitled Space Craft for orbit
[LOG 15:52:01.771] [EL Launchpad] dewarp
[ERR 15:52:01.772] Exception handling event onTimeWarpRateChanged in class AsstVesselModule:System.NullReferenceException: 
  at (wrapper managed-to-native) UnityEngine.MonoBehaviour:StartCoroutine_Auto (System.Collections.IEnumerator)
  at UnityEngine.MonoBehaviour.StartCoroutine (IEnumerator routine) [0x00000] in <filename unknown>:0 
  at PilotAssistant.FlightModules.PilotAssistant.StartCoroutine (IEnumerator routine) [0x00000] in <filename unknown>:0 
  at PilotAssistant.FlightModules.PilotAssistant.VertModeChanged (VertMode newMode, Boolean active, Boolean setTarget) [0x00000] in <filename unknown>:0 
  at PilotAssistant.FlightModules.PilotAssistant.WarpHandler () [0x00000] in <filename unknown>:0 
  at PilotAssistant.FlightModules.AsstVesselModule.WarpHandler () [0x00000] in <filename unknown>:0 
  at EventVoid.Fire () [0x00000] in <filename unknown>:0 

[EXC 15:52:01.776] NullReferenceException
	UnityEngine.MonoBehaviour.StartCoroutine (IEnumerator routine)
	PilotAssistant.FlightModules.PilotAssistant.StartCoroutine (IEnumerator routine)
	PilotAssistant.FlightModules.PilotAssistant.VertModeChanged (VertMode newMode, Boolean active, Boolean setTarget)
	PilotAssistant.FlightModules.PilotAssistant.WarpHandler ()
	PilotAssistant.FlightModules.AsstVesselModule.WarpHandler ()
	EventVoid.Fire ()
	UnityEngine.Debug:LogException(Exception)
	EventVoid:Fire()
	TimeWarp:assumeWarpRate(Single, Boolean, Boolean)
	TimeWarp:setRate(Int32, Boolean, Boolean, Boolean, Boolean)
	TimeWarp:SetRate(Int32, Boolean, Boolean)
	ExtraplanetaryLaunchpads.<DewarpAndBuildCraft>c__Iterator0:MoveNext()
	UnityEngine.MonoBehaviour:StartCoroutine(IEnumerator)
	ExtraplanetaryLaunchpads.ExBuildControl:DoWork_Build(Double)
	ExtraplanetaryLaunchpads.ExBuildControl:DoWork(Double)
	ExtraplanetaryLaunchpads.ExWorkshop:FixedUpdate()
[LOG 15:52:02.767] Unpacking Untitled Space Craft
[LOG 15:52:04.860] 10/14/2017 3:52:04 PM,KerbalAlarmClock,Toggle Changed:True
[EXC 15:52:07.341] MissingMethodException: Method not found: 'Vessel.GetHeightFromTerrain'.
	ExtraplanetaryLaunchpads.ExBuildWindow.FinalizeButton ()
	ExtraplanetaryLaunchpads.ExBuildWindow.WindowGUI (Int32 windowID)
	UnityEngine.GUILayout+LayoutedWindow.DoWindow (Int32 windowID)
	UnityEngine.GUI.CallWindowDelegate (UnityEngine.WindowFunction func, Int32 id, UnityEngine.GUISkin _skin, Int32 forceRect, Single width, Single height, UnityEngine.GUIStyle style)
```

**Comment by chengxinlun** [comment]
Can reproduce with only EL and its requirements. Same exception caused failure.

KSP version: v1.3.1
EL and other requirements: all newest

**Comment by taniwha** [comment]
Ok, yeah, looks like I need to look into what happened to GetHeightFromTerrain

**Comment by taniwha** [comment]
Ah, a simple recompile takes care of it (new default param)

**Comment by taniwha** [comment]
Fixed with the release of 1.3.1 (I guess d0d3a09)
